source/app/static/assets/js/iris/manage_objects.js

```diff
@@ -69,6 +68,23 @@ $('#assets_table').dataTable( {
                     if (type === 'display') { data = sanitizeHTML(data);}
                     return data;
                 }
+            },
+            {
+                "data": "asset_icon_not_compromised",
+                "render": function ( data, type, row ) {
+                    if (type === 'display') { data = sanitizeHTML(data);}
+                    path = "/static/assets/img/graph/"+data
```

Comment:
@whikernel my initial implementation actually included a new persistent volume, my only issue was that it was outside of the apps webroot but this can probably be fixed if the volume is just inside that directory. 
I will work on a persistent implementation.

Reply:
Fixed